lib/onEvent/message.js

```diff
@@ -40,34 +86,38 @@ function checkMessageForCommand (msg, isEdit) {
 	if (isMention(msg.text.split(' '))) {
```

Comment:
why aren't we just doing the check? I dont see the need for an internal function.